### DISCUSSION TITLE
Azure Active Directory B2C (AAD B2C) code sample

### DISCUSSION THREAD

**Discussion by Dailoi2202** [refinedev/refine#5388] [Q&A] [answered]
Has anyone tried to use azure active directory for redirect single sign on, i have seen this link https://refine.dev/docs/advanced-tutorials/auth/azure-ad/#configure-the-msalprovider-component but i couldn't find its code sample

**Comment by BatuhanW** [comment, marked as answer]
Hello @Dailoi2202, we don't have a full example for this one. Still, this example is there to give you an idea.

We can also provide full Azure integration as a part of our enterprise package. You can check it here: https://refine.dev/enterprise/
